### PR TITLE
(DEV-3848) Fixes "How To Pray" Navigation

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Currently navigating to the "How to Pray" content is broken. I added a field to the PrayerRequest API that will give us that content so we can do all sorts of things with it.

### How do I test this PR?

- `yarn test`
- click the "How to Pray" button in the app

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-data-prayer/src/prayer-requests/data-source.js [hunk 1]
```diff
@@ -96,6 +96,13 @@ export default class PrayerRequest extends RockApolloDataSource {
     return this.sortPrayers(prayers);
   };
 
+  getPrayerHelpContent = async () => {
+    const {
+      dataSources: { ContentItem },
+    } = this.context;
+    return ContentItem.getFromId(ROCK_MAPPINGS.PRAYER_HELP_CONTENT_ID);
```
Comment (cdanwards) [inline packages/apollos-data-prayer/src/prayer-requests/data-source.js:103]: 💥

**Review by cdanwards — APPROVED** [review]
Works and tests run! Nice!

**Comment by IsaacHardy** [comment]
@redreceipt If this gets merged before the GraphQL naming PR, you might need to update the naming in your queries in either this PR or that one again.

File: packages/apollos-data-prayer/src/testMocks.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+export class AuthMock {
```
Comment (IsaacHardy) [inline packages/apollos-data-prayer/src/testMocks.js:1]: 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

File: packages/newspringchurchapp/src/prayer/data/queries/getCampusPrayerRequests.js [hunk 3]
```diff
@@ -18,6 +18,10 @@ export default gql`
         id
         name
       }
+      prayerHelpContent {
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/prayer/data/queries/getCampusPrayerRequests.js:21]: I think I'd like to see this queried above these components (in their parent) and passed down to each list of prayers. That is an easy win for less queries.

**Review by IsaacHardy — CHANGES_REQUESTED** [review]

File: packages/newspringchurchapp/src/prayer/data/queries/getCampusPrayerRequests.js (hunk 3 above)
Comment (redreceipt) [reply packages/newspringchurchapp/src/prayer/data/queries/getCampusPrayerRequests.js:21]: ...yea. This method is what was suggested but I think I agree with you. I had the same thought with not as much conviction.

**Comment by redreceipt** [comment]
closing this one. will revisit if we definitely want to keep the "How to Pray" stuff

**Pull request closed without merging**